.github/workflows/deploy_pip_page.yml

```diff
@@ -41,6 +44,45 @@ jobs:
           sed -i.bak 's/\/llvm\/eudsl/https:\/\/github.com\/llvm\/eudsl/g' index.html
           mkdir -p page && mv index.html page
 
+      - name: Fetch latest WASM wheel
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          mkdir -p page/static
+
+          MLIR_PYTHON_WHEEL_NAME=$(gh release view mlir-python-bindings --repo llvm/eudsl --json 'assets' -q '[.assets[] | select(.name | contains("cp312-pyodide")) | .name] | last')
+          MLIR_PYTHON_WHEEL_URL=$(gh release view mlir-python-bindings --repo llvm/eudsl --json 'assets' -q '[.assets[] | select(.name | contains("cp312-pyodide")) | .url] | last')
+          echo "Downloading ${MLIR_PYTHON_WHEEL_NAME} from ${MLIR_PYTHON_WHEEL_URL}"
+          wget -O page/static/${MLIR_PYTHON_WHEEL_NAME} "${MLIR_PYTHON_WHEEL_URL}"
+          echo "MLIR_PYTHON_WHEEL_NAME=$MLIR_PYTHON_WHEEL_NAME" >> $GITHUB_ENV
+
+      - name: Create WASM console page
+        run: |
+          mkdir -p page/console
+
+          cp pages/console/index.template.html page/console/index.html
+          sed -i 's/{{PYODIDE_VERSION}}/v0.27.7/g' page/console/index.html
```

Comment:
why do you need a template for this? for future versions or something? i think we can just hardcore it...

Reply:
Ahh yeah for pyodide  version we can hardcode it.